test-framework/junit5-component/src/test/java/io/quarkus/test/component/declarative/InjectMockTest.java

```diff
@@ -1,32 +0,0 @@
-package io.quarkus.test.component.declarative;
-
-import static org.junit.jupiter.api.Assertions.assertEquals;
-
-import jakarta.inject.Inject;
-
-import org.junit.jupiter.api.Test;
-import org.mockito.Mockito;
-
-import io.quarkus.test.InjectMock;
-import io.quarkus.test.component.QuarkusComponentTest;
-import io.quarkus.test.component.TestConfigProperty;
-import io.quarkus.test.component.beans.Charlie;
-import io.quarkus.test.component.beans.MyComponent;
-
-@QuarkusComponentTest(MyComponent.class)
-@TestConfigProperty(key = "foo", value = "BAR")
-public class InjectMockTest {
```

Comment:
Why delete this class?

Reply:
It's related to the first commit. This test was originally testing the compatibility with the deprecated `@io.quarkus.test.junit.mockito.InjectMock` which is not present in the codebase anymore.

Reply:
Ah, I see `DependencyMockingTest` does pretty much the same thing. This one is good to go then. Thanks!